oracle/src/sender.js

```diff
@@ -167,7 +167,7 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue, channel }) {
 
     if (failedTx.length) {
       logger.info(`Sending ${failedTx.length} Failed Tx to Queue`)
-      await sendToQueue(failedTx)
+      await scheduleForRetry(failedTx, msg.properties.headers['x-retries'])
```

Comment:
Does it make sense to have a test for this functionality?

Reply:
Added unit tests https://github.com/poanetwork/tokenbridge/pull/103/commits/ae6692abcd20b45a4bdb0cb67928feb316e27746